fastlane/metadata/android/zh-CN/full_description.txt

```diff
@@ -0,0 +1,5 @@
+<p>臭名昭著的 OAndBackup 的分叉，目的在于将此应用带入 2020，添加对最新版本 Android 的支持并更新其外观。</p>
```

Comment:
What's the meaning of `infamous`? It's not famous or it has terrible reputation?

Reply:
Bad reputation — OAB hardly ever worked

Reply:
Thanks! Then my translation is correct.